buildall

```diff
@@ -13,19 +14,61 @@ GCR_BASENAME=gcr.io/bitnami-containers/minideb
 
 mkdir -p build
 
+import() {
+    SOURCE=$1
+    TIMESTAMP=$2
+    ./import "$SOURCE" "$TIMESTAMP"
+}
+
+log() {
+    echo "$@" >&2
+}
+
 for DIST in $DISTS; do
    [ -f debootstrap/$DIST ] || (echo "buildall: Unknown distribution: $DIST" && exit 1)
-   echo "============================================"
-   echo "Building $BASENAME:$DIST"
-   echo "============================================"
+   current_ts="$(date -u +%Y-%m-%dT%H:%M:%S.%NZ)"
+   if docker pull $BASENAME:$DIST > /dev/null; then
+       target_ts="$(docker inspect $BASENAME:$DIST | jq --raw-output ".[0].Created")"
```

Comment:
It's not the pulled image timestamp if we didn't pull, I chose "target" as it is the one we are aiming for.